CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## Unreleased
+* Add new experimental rule `no-empty-file` for all code styles. Kotlin and Kotlin Scription files may not be empty.
```

Comment:
Sorry, you did as I asked and put the remark in the "Unreleased" section. I should have been more precise and asking you to put it in the "### added" section of "## Unreleased".

Reply:
Never mind, thank you for sharing my mistake 🙏🏻